docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
 ```
 
-The rule in the example is named `spec`, which indicates compliance with the OpenAPI spec. You can also manually add problems that should be ignored to specific rules.
+The built-in rule in our example is `spec`, which indicates compliance with the OpenAPI Specification. You can also manually add to specific rules problems that should be ignored.
 
-### Skip preprocessor or rule
+### How to open the help
 
-You may want to skip specific preprocessors or rules upon running the command.
+This option works with any command, not just `lint`.
 
-```bash Skip preprocessors
-openapi lint --skip-preprocessor=discriminator-mapping-to-one-of,another-example
+```bash Command
+redocly lint --help
 ```
 
-```bash Skip rules
-openapi lint --skip-rule=no-sibling-refs,no-parent-tags
+```
+cli.js lint [entrypoints...]
+
+Lint definition.
+
+Positionals:
+  entrypoints                                              [array] [default: []]
+
+Options:
+  --version               Show version number.                         [boolean]
+  --help                  Show help.                                   [boolean]
+  --format                Use a specific output format.
+  [choices: "stylish", "codeframe", "json", "checkstyle"] [default: "codeframe"]
+  --max-problems          Reduce output to max N problems.
+                                                         [number] [default: 100]
+  --generate-ignore-file  Generate ignore file.                        [boolean]
+  --skip-rule             Ignore certain rules.                          [array]
+  --skip-preprocessor     Ignore certain preprocessors.                  [array]
+  --config                Specify path to the config file.              [string]
+  --extends               Override extends configurations (defaults or config
+                          file settings).                                [array]
 ```
 
-:::success Tip
-To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
-:::
+### How to limit issues shown in the output
+
+This option lets you limit the number of detected issues (errors and warnings) shown in the output. If issues go over the threshold, the overflow is hidden in a spoiler.
+
+```bash Command
+redocly lint --max-problems 125
+redocly lint mydefinition.yaml --max-problems 125
+```
+
+```bash Output
+...
+< ... 2 more problems hidden > increase with `--max-problems N`
+```
+
+### How to skip a rule or preprocessor during linting
+
+You can use `--skip-rule` to ignore any [custom rules](./resources/custom-rules.md) or [built-in rules](./resources/built-in-rules.md). In this example, we will skip the `spec` and `operation-4xx-response` built-in rules:
+
+```bash
+redocly lint openapi.yaml --skip-rule=spec --skip-rule=operation-4xx-response
+```
+
+Want to know how to skip a preprocessor? Because we don't recommend using preprocessors, you'll understand why we're choosing to skip that advice.
```

Comment:
suggestion: This is clever :smile: but I think we'll be better off not mentioning preprocessors at all. The only information we're giving the user here is that we don't recommend preprocessors. We don't say why, or what will happen if they use (or skip) them, or anything else of value. I'd suggest removing "preprocessor" from the heading and just make this section about skipping rules. Then you don't need this outro anymore.